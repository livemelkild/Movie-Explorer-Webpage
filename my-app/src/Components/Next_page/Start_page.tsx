import React, { useEffect } from "react";
import 'materialize-css/dist/css/materialize.min.css'
import M from 'materialize-css';
import {Link} from "react-router-dom";
import "./Start_page.tsx";


function Start_page(){
  const test = () => {
    var elems = document.querySelectorAll('.slider');
    var instances = M.Slider.init(elems);
  };
  //document.addEventListener("DOMContentLoaded",test);
  
  useEffect( () => {
    test();
  },[]);

    return(
        <nav>
        <div className="nav-wrapper green darken-3">
        <a href="#!" className="brand-logo center">Movie Expolrer</a>
          <ul id="nav-mobile" className="right hide-on-med-and-down">
          <Link className = "start_link" to="/next">
            <li><a> <b>Search</b></a></li>
          </Link>
          </ul>
        </div>
      <div>
        <h1 className="homepage"> </h1>
        <div className="slider">
        <ul className="slides">
        <li>
        <img src="https://secure.static.tumblr.com/4d3dd0aabf6c52c57afc18879c169ae1/riqitkz/9nhn15y8u/tumblr_static_2012_movies_collage.jpg" /> 
        <div className="caption center-align">
          <h3><b>Find your favourite movie</b></h3>
          <h5 className="light grey-text text-lighten-3"></h5>
        </div>
      </li>
      
      <li>
        <img src="https://www.barnevakten.no/wp-content/uploads/2010/11/harry-potter-deathly-hallows-1-1.jpg" /> 
        <div className="caption right-align ">
          <h3> <b> Comedy, Action, Magic? </b></h3>
          <h5 className="light grey-text text-lighten-3"></h5>
        </div>
      </li>
      <li>
      <img src="https://s3-us-west-2.amazonaws.com/flx-editorial-wordpress/wp-content/uploads/2018/03/13153742/RT_300EssentialMovies_700X250.jpg" /> 
        <div className="caption center-align">
          <h3> <b>No limits... </b> </h3>
          <h5 className="light grey-text text-lighten-3"></h5>
        </div>
        </li>
        </ul>     
    </div>
    </div>

    </nav>
   

    )
}

export default Start_page;