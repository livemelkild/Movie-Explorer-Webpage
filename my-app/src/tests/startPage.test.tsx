import React from 'react';
import { render } from '@testing-library/react';
import { Provider } from 'react-redux'
import renderer from 'react-test-renderer'
import { idText } from 'typescript';
import Start_page from '../Components/Next_page/Start_page';
import { BrowserRouter } from 'react-router-dom';


test('Page snapshot test', () => {
    const page = renderer.create(<BrowserRouter><Start_page/></BrowserRouter>) 
    expect(page).toMatchSnapshot()

})