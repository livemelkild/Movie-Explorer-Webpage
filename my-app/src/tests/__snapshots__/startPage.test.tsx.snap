// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Page snapshot test 1`] = `
<nav>
  <div
    className="nav-wrapper green darken-3"
  >
    <a
      className="brand-logo center"
      href="#!"
    >
      Movie Explorer
    </a>
    <ul
      className="right hide-on-med-and-down"
      id="nav-mobile"
    >
      <a
        className="start_link"
        href="/next"
        onClick={[Function]}
      >
        <li>
          <a>
             
            <b>
              Search
            </b>
          </a>
        </li>
      </a>
    </ul>
  </div>
  <div>
    <h1
      className="homepage"
    >
       
    </h1>
    <div
      className="slider"
    >
      <ul
        className="slides"
      >
        <li>
          <img
            src="https://secure.static.tumblr.com/4d3dd0aabf6c52c57afc18879c169ae1/riqitkz/9nhn15y8u/tumblr_static_2012_movies_collage.jpg"
          />
          <div
            className="caption center-align"
          >
            <h3>
              <b>
                Find your favourite movie
              </b>
            </h3>
            <h5
              className="light grey-text text-lighten-3"
            />
          </div>
        </li>
        <li>
          <img
            src="https://www.barnevakten.no/wp-content/uploads/2010/11/harry-potter-deathly-hallows-1-1.jpg"
          />
          <div
            className="caption right-align "
          >
            <h3>
               
              <b>
                 Comedy, Action, Magic? 
              </b>
            </h3>
            <h5
              className="light grey-text text-lighten-3"
            />
          </div>
        </li>
        <li>
          <img
            src="https://s3-us-west-2.amazonaws.com/flx-editorial-wordpress/wp-content/uploads/2018/03/13153742/RT_300EssentialMovies_700X250.jpg"
          />
          <div
            className="caption center-align"
          >
            <h3>
               
              <b>
                No limits... 
              </b>
               
            </h3>
            <h5
              className="light grey-text text-lighten-3"
            />
          </div>
        </li>
      </ul>
    </div>
  </div>
</nav>
`;
